music_assistant/server/providers/tidal/helpers.py

```diff
@@ -74,7 +77,12 @@ def inner() -> TidalArtist:
         try:
             artist_obj = TidalArtist(session, prov_artist_id)
         except HTTPError as err:
-            raise MediaNotFoundError(f"Artist {prov_artist_id} not found") from err
+            if err.response.status_code == 404:
+                raise MediaNotFoundError(f"Artist {prov_artist_id} not found") from err
+            else:
+                LOGGER.error(
+                    f"HTTPError {err.response.status_code} for Artist with id: {prov_artist_id}"
+                )
         return artist_obj
```

Comment:
this will raise due to artist_obj not being initialized

Reply:
I feel like I've been here before. I'm getting overly cautious for possible future errors, and introducing my own errors. 
I'll try to return it back to sensibility